### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by pjmartorell** [carrierwaveuploader/carrierwave#2553]
Hi @mshibuya, do you think it would be possible to release and publish a new version including the latest changes replacing `mimemagic` with `marcel` gem?

Thank you

**Comment by oakbow** [comment]
I agree with @pjmartorell 

we have a trouble with mimemagic because of version mismatch.
I wrote the detail of this problem at https://github.com/carrierwaveuploader/carrierwave/issues/2554

I think it is the best way to release a new version.

**Comment by TCCDevelopment** [comment]
I also agree with @pjmartorell

**Comment by yusupss** [comment]
100% agree with @pjmartorell , I think we need the new release asap

**Comment by pedrofurtado** [comment]
Related to: https://github.com/carrierwaveuploader/carrierwave/issues/2554

**Comment by jaredbeck** [comment]
Just a friendly reminder, you can easily specify the `master` branch in your `Gemfile`.

```ruby
# Using `master` until #2551 is released.
gem 'carrierwave',
  github: 'carrierwaveuploader/carrierwave',
  branch: 'master',
  ref: '3f809672dd2d18ee4398a7bba9d8f9effa05023e'
```

In this way, we can take the pressure off of the volunteers responsible for putting together a release. :heart:

**Comment by katelovescode** [comment]
> Just a friendly reminder, you can easily specify the `master` branch in your `Gemfile`.
> 
> ```ruby
> # Using `master` until #2551 is released.
> gem 'carrierwave',
>   github: 'carrierwaveuploader/carrierwave',
>   branch: 'master',
>   ref: '3f809672dd2d18ee4398a7bba9d8f9effa05023e'
> ```
> 
> In this way, we can take the pressure off of the volunteers responsible for putting together a release. ❤️

Unfortunately in the application in which we're using carrierwave, we haven't been approved to use Github as a source 😒 so we have to wait until it's released to rubygems.  ☹️

**Comment by waruboy** [comment]
would love to see this release too, related to mimemagic. Anything I can help with?

**Comment by mshibuya** [comment]
Just released 2.2.1 🚢
